Magento2/Sniffs/Functions/FunctionsDeprecatedWithoutArgumentSniff.php

```diff
@@ -31,15 +31,15 @@ class FunctionsDeprecatedWithoutArgumentSniff implements Sniff
     private const WARNING_CODE = 'FunctionsDeprecatedWithoutArgument';
 
     /**
-     * Deprecated functions without argument.
+     * Deprecated functions without argument
      *
      * @var array
      */
-    private const FUNCTIONS_LIST = [
-        'mb_check_encoding',
-        'get_class',
-        'get_parent_class',
-        'get_called_class'
+    private const DEPRECATED_FUNCTIONS_AND_FIXES = [
+        'mb_check_encoding' => 'null',
+        'get_class' => '$this',
+        'get_parent_class' => '$this',
+        'get_called_class' => '$this'
```

Comment:
Using arguments with get_called_class leads to the following error: `Fatal error: Uncaught ArgumentCountError: get_called_class() expects exactly 0 arguments, 1 given`
Please remove this function from the list

Reply:
That's exactly what I was suspecting...